images/airflow/2.10.3/python/mwaa/entrypoint.py

```diff
@@ -790,6 +790,6 @@ async def main() -> None:
 
 if __name__ == "__main__":
     asyncio.run(main())
-else:
+elif os.environ["TESTING_MODE"] != "true":
```

Comment:
Here, TESTING_MODE is becoming a reserved env variable but it is possible that the customer also defines something similar for there own experimentation. So, it's a good idea to have all env variables defined by us with some MWAA specific naming, eg: `MWAA__CORE__TESTING_MODE`.